hm_pyhelper/tests/test_miner_json_rpc.py

```diff
@@ -0,0 +1,244 @@
+import unittest
+import mock
+import responses
+import requests
+from hm_pyhelper.miner_json_rpc import MinerClient
+from hm_pyhelper.miner_json_rpc.exceptions import MinerRegionUnset
+from hm_pyhelper.miner_json_rpc.exceptions import MinerMalformedURL
+from hm_pyhelper.miner_json_rpc.exceptions import MinerConnectionError
+
+BASE_URL = 'http://helium-miner:4467'
+
+
+@responses.activate
+def response_result(data, status):
+    url = "https://fake_url"
+    responses.add(responses.POST, url, json=data, status=status)
+    resp = requests.post(url)
+    print(resp.json())
```

Comment:
remove print

Reply:
it is in test code. Probably can be left as debug statement.